docs/topics/signals.txt

```diff
@@ -189,6 +189,20 @@ In some circumstances, the code connecting receivers to signals may run
 multiple times. This can cause your receiver function to be registered more
 than once, and thus called multiple times for a single signal event.
 
+As Django has evolved, this issue has generally less risk of presenting itself.
+Before ``AppConfig.ready`` was introduced to Django, it was possible that
+during the Django setup process during normal operation or from a management
+command, the handlers.py file would be imported multiple times, and thus,
+the signal handler would be registered each time.
+
+Another (less common but still possible) occurrence is if in your application
+code you import your handlers.py module from somewhere in your project because
+you need to call or run logic against a handler function, and thus the signal
+registration would run at least one more time per-import instance. As projects
```

Comment:
```suggestion
you need to call or run logic against a handler function. This will result in the signal
registration being run at least one more time per-import instance. As projects
```

Reply:
Hopefully the new wording is clearer, but resolving since the sentence has changed.